frontend/app-development/layout/PageHeader.tsx

```diff
@@ -36,27 +36,41 @@ const subMenuContent = () => {
   );
 };
 
-const buttonActions = (org: string, app: string): AltinnButtonActionItem[] => {
-  const actions = [
-    {
-      title: 'top_menu.preview',
-      path: previewPath,
-      menuKey: TopBarMenu.Preview,
-      buttonVariant: ButtonVariant.Outline,
-      headerButtonsClasses: classes.previewButton,
-      handleClick: () => window.open(previewPath(org, app), '_blank'),
-      inBeta: true,
-    },
-    {
-      title: 'top_menu.deploy',
-      path: publiserPath,
-      menuKey: TopBarMenu.Deploy,
-      buttonVariant: ButtonVariant.Outline,
-      headerButtonsClasses: undefined,
-      handleClick: () => (window.location.href = publiserPath(org, app)),
-    },
-  ];
-  return actions;
+export const buttonActions = (org: string, app: string): AltinnButtonActionItem[] => {
+  if (window.location.pathname.includes('preview')) {
+    const previewAction = [
```

Comment:
Preview is a separate app, so here you should probably define the preview header content in `app-preview`, not in `app-development`. That saves you from having to do the check on pathname as well.

Reply:
done